src/Helmich/TypoScriptParser/Tokenizer/Tokenizer.php

```diff
@@ -40,7 +40,7 @@ class Tokenizer implements TokenizerInterface
     const TOKEN_INCLUDE_STATEMENT = ',^
         <INCLUDE_TYPOSCRIPT:\s+
         source="(?<type>FILE|DIR):(?<filename>[^"]+)"
-        (?:\s+extension="(?<extension>[^"]+)")?
+        (?:\s+extensions="(?<extensions>[^"]+)")?
```

Comment:
I'm pretty sure that the pattern name `extension` is used somewhere in the `Parser` class when building the AST; when renaming the pattern, the parser would need to be changed, too.